wiki/Beatmap/Overall_difficulty/en.md

```diff
@@ -16,7 +16,7 @@ tags:
 
 Higher OD values mean shorter time windows to clear [hit objects](/wiki/Hit_object), both in general and in terms of getting high [score](/wiki/Gameplay/Score) values. The maximum allowed hit error for each hit object in osu! and osu!mania, centred on the hit object's correct time, is defined by the tables below.
 
-Note that in the stable version of osu!, hit windows in osu! and osu!taiko can effectively be up to 0.5 ms shorter on both sides than what the formulae suggest, and in osu!mania they can be up to 0.5 ms longer on both sides. This is because in osu! and osu!taiko, a hit is considered inside a hit window if `hit error < round(hit window)`, while in osu!mania it is considered inside if `hit error <= round(hit window)`.
+Note that in the stable version of osu!, hit windows in osu! and osu!taiko can effectively be up to 0.5 ms shorter on both sides than what the formula suggest, and in osu!mania they can be up to 0.5 ms longer on both sides. This is because in osu! and osu!taiko, a hit is considered inside a hit window if `hit error < round(hit window)`, while in osu!mania it is considered inside if `hit error <= round(hit window)`.
```

Comment:
`formulae` is actually correct tho, as it's an alternative spelling of `formulas` (http://writingexplained.org/formulas-or-formulae-correct)

...although yeah i looked around the wiki and other articles such as http://osu.ppy.sh/wiki/en/Beatmap/Approach_rate and http://osu.ppy.sh/wiki/en/Client/Beatmap_editor/Song_Setup went on to use `formulas` instead

Reply:
hmm TIL, should we change it to `formulas` instead?